backend/src/main/scala/cromwell/backend/async/AsyncBackendJobExecutionActor.scala

```diff
@@ -72,8 +72,8 @@ trait AsyncBackendJobExecutionActor { this: Actor with ActorLogging =>
     case PollResponseReceived(handle) if handle.isDone => self ! Finish(handle)
     case PollResponseReceived(handle) =>
       context.system.scheduler.scheduleOnce(pollBackOff.backoffMillis.millis, self, IssuePollRequest(handle))
-    case Finish(SuccessfulExecutionHandle(outputs, returnCode, resultsClonedFrom)) =>
-      completionPromise.success(SucceededResponse(jobDescriptor.key, Some(returnCode), outputs))
+    case Finish(SuccessfulExecutionHandle(outputs, returnCode, callOutputFiles, resultsClonedFrom)) =>
+      completionPromise.success(SucceededResponse(jobDescriptor.key, Some(returnCode), outputs, Some(callOutputFiles)))
```

Comment:
It took me a while to figure out `callOutputFiles` actually refers to the job detritus (not that I love that name, but at least I understand it's different from the call/job outputs)